lua/fidget/integration/xcodebuild-nvim.lua

```diff
@@ -0,0 +1,86 @@
+local M = {}
+
+-- Only register one callback, so setup() can be called multiple times.
+local already_subscribed = false
+
+---@options integration.xcodebuild-nvim [[
+--- xcodebuild.nvim integration
+M.options = {
+  --- Integrate with wojciech-kulik/xcodebuild.nvim (if installed)
+  ---
+  --- Dynamically offset Fidget's notifications window when the Test Explorer window
+  --- is open on the right side + the Fidget window is "editor"-relative.
+  ---
+  ---@type boolean
+  enable = true,
+}
+---@options ]]
+
+require("fidget.options").declare(
+  M,
```

Comment:
nit: is this formatted using stylua? It's inconsistent with the style in the other modules.

Reply:
yeap, I will fix it